tests/parsers/perkin_elmer_envision/testdata/example01_missing_header.csv

```diff
@@ -0,0 +1,217 @@
+Plate information 
+Plate,Repeat,Barcode,Measured height,Chamber temperature at start,Chamber temperature at end,Humidity at start,Humidity at end,Ambient temperature at start,Ambient temperature at end,Formula,Measurement date,
+1,1,"=""""",N/A,N/A,N/A,N/A,N/A,N/A,N/A,Calc 1: General = (X / Y) * Z where X = AC HTRF Laser [Eu](1) channel 1 window 1  Y = AC HTRF Laser [Eu](1) channel 2 window 1  Z = 10000,10/13/2022 3:08:06 PM,
+
+Background information
+Plate,Label,Result,Signal,Flashes/Time,Meastime,MeasInfo,
+1,AC HTRF Laser [Eu],0,162,50,00:00:00.000,De=1st Ex=Top Em=Top Wdw=1 (14),
+1,AC HTRF Laser [Eu],0,260,50,00:00:00.000,De=2nd Ex=Top Em=Top Wdw=1 (142),
+
+Calculated results: Calc 1: General = (X / Y) * Z where X = AC HTRF Laser [Eu](1) channel 1 window 1  Y = AC HTRF Laser [Eu](1) channel 2 window 1  Z = 10000
+,01,02,03,04,05,06,07,08,09,10,11,12,
+A,3912.12920565,3923.92517971,3948.45245355,,3933.03506298,4061.92827247,3996.61746992,,,,,,
+B,2385.72193555,2384.42487617,2396.10683014,,2390.61028269,2396.68321027,2424.70697585,,,,,,
+C,1497.6939597,1488.29789767,1474.23429335,,1513.01294815,1487.52148035,1471.02712185,,,,,,
+D,1004.96752006,997.698002862,984.957281601,,1017.18554813,968.985790113,1005.07749543,,,,,,
+E,707.203907204,697.464104756,703.487537674,,726.238246392,701.399029629,721.275009761,,,,,,
+F,562.804988252,557.881619938,559.664129137,,573.562254074,567.634735212,563.983204482,,,,,,
+G,481.189418042,482.180558541,480.169042819,,478.930200648,482.450907876,491.042828331,,,,,,
+H,403.155127082,414.191684058,399.233942251,,403.302222012,406.559451108,409.025761641,,,,,,
+
+Plate,Repeat,Barcode,Measured height,Chamber temperature at start,Chamber temperature at end,Humidity at start,Humidity at end,Ambient temperature at start,Ambient temperature at end,Group,Label,ScanX,ScanY,Measinfo,Kinetics,Measurement date,
+1,1,"=""""",11.9,23.17,23.19,100,100,23.47,23.59,1,AC HTRF Laser [Eu](1),0,0,De=1st Ex=Top Em=Top Wdw=1 (14),0,10/13/2022 3:08:06 PM,
+
+Background information
+Plate,Label,Result,Signal,Flashes/Time,Meastime,MeasInfo,
+1,AC HTRF Laser [Eu],0,162,50,00:00:00.000,De=1st Ex=Top Em=Top Wdw=1 (14),
+1,AC HTRF Laser [Eu],0,260,50,00:00:00.000,De=2nd Ex=Top Em=Top Wdw=1 (142),
+
+Results for AC HTRF Laser [Eu](1) - channel 1 window 1 (Counts)
+,01,02,03,04,05,06,07,08,09,10,11,12,
+A,31441,31278,31727,,32350,32415,32847,,,,,,
+B,19603,19063,19621,,19543,19799,20087,,,,,,
+C,12762,12502,12582,,12842,13071,12914,,,,,,
+D,8153,8018,8093,,8387,8142,8294,,,,,,
+E,5792,5630,5882,,6063,5956,6096,,,,,,
+F,4671,4477,4639,,4854,4728,4822,,,,,,
+G,3887,3814,3954,,3979,4044,4139,,,,,,
+H,3266,3354,3252,,3405,3342,3339,,,,,,
+
+Plate information 

```

Comment:
Can we drop the rest of this file, in the interest of keeping this testing data as succinct as possible

Reply:
Fixed in [d71594e](https://github.com/Benchling-Open-Source/allotropy/pull/72/commits/d71594e2935c91fd5bb0383205a1e1ad5953c605)